gpustack/policies/candidate_selectors/gguf_resource_fit_selector.py

```diff
@@ -1927,7 +1940,7 @@ async def _generate_combinations_for_worker_with_rpcs_with_selected_gpu_ids(  #
                 self._event_collector.add(
                     EventLevelEnum.ERROR,
                     EVENT_ACTION_DISTRIBUTED_DEPLOYMENT,
-                    f"Selected GPU {key} lacks enough VRAM to serve as an RPC server. At least {byte_to_gib(rpc_at_least_vram)} GiB is required.",
+                    f"- The model in RPC requires approximately {byte_to_gib(rpc_at_least_vram)} GiB VRAM.\n- Selected GPU {key} lacks enough VRAM to serve as an RPC server.",
```

Comment:
Can we have a helper function like `build_list_message(List[str])` to handle prefix hyphens, newlines, etc.